<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IndentedTextExportWidget</class>
 <widget class="QWidget" name="IndentedTextExportWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>311</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>468</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Indented Text Export</string>
  </property>
  <widget class="QCheckBox" name="checkBoxPreview">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Show &amp;preview</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_OriginX">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_OriginY">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99999</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_CellWidth">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="maximum">
    <double>1200.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_CellHeight">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="maximum">
    <double>1200.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Input Text Origin (pixels)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>85</y>
     <width>20</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;X</string>
   </property>
   <property name="buddy">
    <cstring>spinBox_OriginX</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>85</y>
     <width>20</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Y</string>
   </property>
   <property name="buddy">
    <cstring>spinBox_OriginY</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Input text cell size (pixels)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>135</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Width</string>
   </property>
   <property name="buddy">
    <cstring>doubleSpinBox_CellWidth</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>135</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Height</string>
   </property>
   <property name="buddy">
    <cstring>doubleSpinBox_CellHeight</cstring>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_OriginCompute">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Estimate O&amp;rigin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_CellCompute">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Estimate &amp;Size</string>
   </property>
  </widget>
  <widget class="QLabel" name="helpLabel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Align scan to original text so whitespace is accurate. Assumes (but does not require) monospace font.&lt;/p&gt;&lt;p&gt;Manually fine tune the Estimated values using the preview.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/extra-theme-icons/help-hint.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>451</width>
     <height>111</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>These settings do not affect the result, but can be used to align the displayed result with the scan.</string>
   </property>
   <property name="title">
    <string>Preview Settings</string>
   </property>
   <widget class="QFontComboBox" name="comboBox_FontFamily">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
    <property name="currentText">
     <string>Monospace</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_FontSize">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="suffix">
     <string> pt</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinBox_FontStretch">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="toolTip">
     <string>Fine tune preview alignment with scan image.</string>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>200.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>100.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="labelFontScaling">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Font Stretch</string>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
    <property name="buddy">
     <cstring>spinBox_FontStretch</cstring>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_GuideBars">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Show &amp;Guide Bars</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Fon&amp;t</string>
    </property>
    <property name="buddy">
     <cstring>comboBox_FontFamily</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Siz&amp;e</string>
    </property>
    <property name="buddy">
     <cstring>spinBox_FontSize</cstring>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_EditorGrid">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Copy Editor Grid</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>checkBoxPreview</tabstop>
  <tabstop>pushButton_EditorGrid</tabstop>
  <tabstop>spinBox_OriginX</tabstop>
  <tabstop>spinBox_OriginY</tabstop>
  <tabstop>pushButton_OriginCompute</tabstop>
  <tabstop>doubleSpinBox_CellWidth</tabstop>
  <tabstop>doubleSpinBox_CellHeight</tabstop>
  <tabstop>pushButton_CellCompute</tabstop>
  <tabstop>comboBox_FontFamily</tabstop>
  <tabstop>spinBox_FontSize</tabstop>
  <tabstop>spinBox_FontStretch</tabstop>
  <tabstop>checkBox_GuideBars</tabstop>
 </tabstops>
 <resources>
  <include location="gimagereader.qrc"/>
 </resources>
 <connections/>
</ui>
